hw_3.Functions.py

```diff
@@ -0,0 +1,158 @@
+# 1) Write a function that emulates the game "rock, scissors, paper"
+# At the entrance, your function accepts your version printed from the console, the computer makes a decision randomly.
+import random
+def game_rockscissorspaper():
+    def determine_the_winner(var1, var2):
+        if (var1 == 'rock' and var2 == 'scissors') or + \
+                (var1 == 'paper' and var2 == 'rock') or + \
+                (var1 == 'scissors' and var2 == 'paper'):
+            print('You won')
+        elif var1 == var2:
+            print('Draw')
+        else:
+            print('Computer won')
+
+    var_list = ('rock', 'scissors', 'paper')
+    var1 = input('Input rock, scissors or paper: ')
```

Comment:
name `var1` and `var2` it's not a good choice.

Reply:
What about player1, player2 ?